### PR TITLE
Add more unit tests

### PR DESCRIPTION
**Pull request by milmazz**
This PR include the following:

* More unit tests
* Include an image logo as a fixture
* Proposes to use [ExCoveralls](https://github.com/parroty/excoveralls) as a coverage report tool.

/cc @josevalim

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -2,18 +2,21 @@ defmodule ExDoc.Mixfile do
...
+     source_url:        "https://github.com/elixir-lang/ex_doc/",
+     test_coverage:     [tool: ExCoveralls],
+     preferred_cli_env: [coveralls: :test]]
   end
 
   defp deps do
     [{:earmark, "~> 0.1.17 or ~> 0.2", optional: true},
-     {:markdown, github: "devinus/markdown", only: [:test]}]
+     {:markdown, github: "devinus/markdown", only: [:test]},
+     {:excoveralls, "~> 0.3", only: :test}]
```
Comment (josevalim) [inline mix.exs]: `only: [:test]` to be consistent with the line above?

**Comment by josevalim** [comment]
Let's please have an empty png file. There is no need to include the elixir logo in the project. :D

**Comment by milmazz** [comment]
@josevalim Done :)

**Comment by josevalim** [comment]
:shipit:

**Pull request merged**